bundles/core/org.eclipse.smarthome.core.thing.xml.test/src/test/resources/test-bundle-pool/SystemChannels.bundle/ESH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="SystemChannels"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0"
+        xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 http://eclipse.org/smarthome/schemas/thing-description-1.0.0.xsd">
+
+    <!-- YahooWeather Binding -->
+    <thing-type id="wireless-router">
+        <label>Simple wireless router</label>
+        <description>Simple wireless router</description>
+
+		<channels>
+			<channel id="sigstr" typeId="system.signal-strength" />
+			<channel id="test" typeId="system.my-channel" />
+			<channel id="lowbat" typeId="system.low-battery" />
+		</channels>
+
+    </thing-type> 
+    
+    <!-- System Channel Type -->
+	<channel-type id="my-channel" system="true">
```

Comment:
One question regarding this XML: Is it possible to define a system wide channel types via XML without defining a thing type?